### PR TITLE
chore: update manifests for release 0.0.8

### LINKED ISSUE #37: Release signed binaries
For windows, we need to sign the .exe and ps1 files.

### LINKED ISSUE #120: Volume mount failed on node with taints
**What steps did you take and what happened:**
I am moving my pods to specific nodes with taints and the volume mount failed with the following errer. 

FailedMount: MountVolume.SetUp failed for volume "secrets-store" : kubernetes.io/csi: mounter.SetUpAt failed to get CSI client: driver name secrets-store.csi.k8s.io not found in the list of registered CSI drivers

**What did you expect to happen:**

**Anything else you would like to add:**
This worked before I tried to move the pod to a node with a taint. We want to get all of our custom applications on to separate nodes so we can control scaling better for our applications separate from system level applications such as this.

**Which access mode did you use to access the Azure Key Vault instance:**
[e.g. Service Principal, Pod Identity, User Assigned Managed Identity, System Assigned Managed Identity] Service Principal

**Environment:**

- Secrets Store CSI Driver version: v0.0.10:
- Azure Key Vault provider version: (use the image tag): 0.0.5
- Kubernetes version: 1.16.9:
- Cluster type: (e.g. AKS, aks-engine, etc): AKS

### LINKED ISSUE #121: Pods restart every time the chart is deployed
**What steps did you take and what happened:**
As part of integrating this module with our CI/CD pipeline, we're configuring it to do a helm deployment of the CSI driver chart every time the pipeline is run.  However, I've noticed that the pods restart every single time this is done even when nothing about the configuration changes.

Is there a good reason for this?  At least one of the pods can take several minutes to terminate and restart.

**What did you expect to happen:**
Pods should only restart if changes are required to the deployment.

**Which access mode did you use to access the Azure Key Vault instance:**
Service Principal

**Environment:**
- Secrets Store CSI Driver version: v0.0.11
- Azure Key Vault provider version: 0.0.6
- Kubernetes version: 1.16.6
- Cluster type: AKS

### PR DESCRIPTION
**Pull request by aramase**
**What this PR does / why we need it**:
- Updates helm chart labels to align with the helm best practices

**Which issue(s) this PR fixes** *(optional, using `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when the PR gets merged)*:
Fixes #121
Fixes https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/195
Fixes #120 
Fixes #37 

**Special notes for your reviewer**:

### CODE REVIEW

File: test/bats/azure.bats [hunk 1]
```diff
@@ -64,13 +61,14 @@ setup() {
...
-}
+  run helm install csi charts/csi-secrets-store-provider-azure --namespace dev --set windows.enabled=true \
+      --set secrets-store-csi-driver.windows.enabled=true \
+      --set image.repository=${PROVIDER_TEST_IMAGE} \
+      --set image.tag=${IMAGE_TAG} \
+      --set image.pullPolicy="IfNotPresent" \
+      --dependency-update
 
-@test "install azure provider with e2e image" {
-  yq w -d 1 ${PROVIDER_YAML} "spec.template.spec.containers[0].image" "${PROVIDER_TEST_IMAGE}:${IMAGE_TAG}" \
```
Comment (ritazh) [inline test/bats/azure.bats:72]: why do we want this removed?

File: test/bats/azure.bats [hunk 2]
```diff
@@ -64,13 +61,14 @@ setup() {
...
 @test "install driver helm chart" {
-  run helm install csi-secrets-store ${SECRETS_STORE_CSI_DRIVER_PATH}/charts/secrets-store-csi-driver --namespace dev --set windows.enabled=true
-  assert_success
-}
+  run helm install csi charts/csi-secrets-store-provider-azure --namespace dev --set windows.enabled=true \
+      --set secrets-store-csi-driver.windows.enabled=true \
+      --set image.repository=${PROVIDER_TEST_IMAGE} \
+      --set image.tag=${IMAGE_TAG} \
+      --set image.pullPolicy="IfNotPresent" \
+      --dependency-update
```
Comment (ritazh) [inline test/bats/azure.bats:69]: 👍

File: sample/ingress-controller-tls/README.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ helm install stable/nginx-ingress --generate-name \
 - Set secret type to `kubernetes.io/tls`
 
 ```bash
-$ cat <<EOF | kubectl apply -f -
+$ cat <<EOF | kubectl apply -n ingress-test -f -
```
Comment (ritazh) [inline sample/ingress-controller-tls/README.md:44]: +1

File: test/bats/azure.bats (hunk 1 above)
Comment (aramase) [reply test/bats/azure.bats:72]: The driver and provider are installed using the helm charts, so we no longer need to install using `kubectl apply -f`

Comment (aramase) [reply test/bats/azure.bats:72]: Issue to track testing with deployment yamls - https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/196

Comment (ritazh) [reply test/bats/azure.bats:72]: This issue is being tracked [here](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues/196) to unblock this PR.

**Pull request merged**